CNAME

```diff
@@ -1 +0,0 @@
-ieeespac.ca
```

Comment:
The CNAME record is required for github's DNS to reflect that the *.github.io are alias domains. Please remove this change.